packages/coachmark/src/Coachmark.ts

```diff
@@ -273,7 +273,7 @@ export class Coachmark extends Popover {
         return html`
             <div class="step" role="status">
                 <span aria-live="polite">
-                    ${this.currentStep} of ${this.totalSteps}
+                    <slot name="step-count"></slot>
```

Comment:
```suggestion
                    <slot name="step-count">${this.currentStep} of ${this.totalSteps}</slot>
```
I think we should default to the English delivery here in order to reduce chances of happy path issues.